.github/workflows/validate.yml

```diff
@@ -75,13 +75,15 @@ jobs:
       - uses: actions/download-artifact@v4
         id: download
         with:
-          name: package
+          name: dist
           path: ./dist
-      - run: cmd /r dir /b /a-d dist > files.txt
       - run: |
-          $files = Get-Content files.txt
+          $files=Get-ChildItem dist
           foreach ($file in $files) {
-            pip install dist/$file
+            if ($file.name -like "*-win_amd64.whl") {
```

Comment:
?

Reply:
The reason is that the name depends on the version, it will be e.g. `numerous-0.22.0-py3-none-win_amd64.whl`.

We want to install the expected wheel. Maybe it would be nice to do it with pypiserver instead to test that it selects the right wheel by itself now though...

Reply:
This is actually changed now, since the windows test launches a pypiserver and installs with that to ensure that it works with the specific windows wheel